rfd/0109-cloud-agent-upgrades.md

```diff
@@ -327,6 +327,33 @@ to be changed to follow the same pattern of registering a release channel and in
 by ec2 discovery.
 
 
+### ECS Fargate Agents
+When Teleport Discover flow is used to connect an RDS, a Teleport agent is automatically provisioned using ECS Fargate. The automatic update
+functionality for regular agents depends on systemd or Kubernetes. Fargate agents run in containers, so that means the existing automatic
+update functionality is not compatible with these agents. We need a separate method of keeping these Fargate agents updated.
+
+To keep Farage agents updated, Teleport will run a separate task that checks every 30 minutes for an available update and uses the ECS
```

Comment:
Where is that task going to run? I think we should run it on the Proxy

Reply:
What would be the benefits of running this task on proxy vs auth?

Reply:
We can run this on Auth Service :+1:
My goal was to be explicit about it here, and I usually default to Proxy service